<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestGestionStock</class>
 <widget class="QWidget" name="TestGestionStock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestGestionStock</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Entrée d'un rouleau</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>441</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Rouleaux en stock</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetStock"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRetrait">
        <property name="text">
         <string>Demander retrait</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Rouleaux retenus</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetRouleauxRetenus"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSupprimer">
        <property name="text">
         <string>Supprimer du stock</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Référence</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditReference"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Diametre</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditDiametre"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="pushButtonNouveau">
        <property name="text">
         <string>Ajouter nouveau</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonQuitter">
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonEnregistrer">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCharger">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Charger</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>TestGestionStock</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
